docs/kibana/plugins.md

```diff
@@ -111,6 +111,15 @@ sudo bin/kibana-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/k
 This plugin provides a user interface for using SQL queries to explore your data.
 
 
+#### Trace Analytics
+
+```bash
+sudo bin/kibana-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/kibana-plugins/opendistro-notebooks/opendistro-notebooks-kibana-{{site.odfe_version}}.0.zip
```

Comment:
Also, since the release is now alpha, the kibana plugin will not be in cloudfront instead a github release. https://github.com/opendistro-for-elasticsearch/trace-analytics/releases/download/v1.12.0.0-alpha/trace-analytics-v1.12.0.0-alpha.zip